client/src/components/CardContainer/index.js

```diff
@@ -0,0 +1,3 @@
+import CardContainer from "./CardContainer";
```

Comment:
I think you do not need this file as we have index js in components folder. just remove this file and import your component directly to components/index.js. Finally, add it to exported files as in a screenshot. 
![image](https://user-images.githubusercontent.com/112622259/199594422-a82bc203-c7b4-464c-bc36-97d0418ff6c2.png)